icu4c/source/configure

```diff
@@ -4264,6 +4264,8 @@ fi
 #AC_CHECK_PROG(AUTOCONF, autoconf, autoconf, true)
 #AC_CHECK_PROG(STRIP, strip, strip, true)
 
+
+
```

Comment:
nit: extra whitespace

Reply:
This file is generated, so I assume that we don't have control over the formatting.